### DISCUSSION TITLE
Fails to run

### DISCUSSION THREAD

**Discussion by gostone** [vkbo/novelWriter#583] [Help] [answered]
Hi. This may be my fault, I don't use Python very much. I've installed novelWriter on my MacBook according to the instructions, using **pip install novelWriter**. This completes successfully. When I try to start the program with **python novelWriter.py** it fails with the message: 

.pyenv/versions/3.8.6/bin/python3: can't open file 'novelWriter.py': [Errno 2] No such file or directory

The installation log output to the screen looks like this: 

Installing collected packages: PyQt5-sip, pyqt5, lxml, pyenchant, novelWriter
Successfully installed PyQt5-sip-12.8.1 lxml-4.6.2 novelWriter-1.0.2 pyenchant-3.2.0 pyqt5-5.15.2

Any help greatly appreciated!

**Comment by vkbo** [comment, marked as answer]
The `python novelWriter.py` command is for when you run directly from the source code. When you install with pip, it should make an executable file named `novelWriter` or `novelwriter` on your system. Did you try to just type one of those in the terminal?

You should also install the `pyobjc` package on macOS. It improves the integration with the OS a bit.

**Reply by gostone** [reply]
yes that works thanks. This looks like it's got great promise, but the menus aren't working for me! I click on them and nothing happens.

**Reply by vkbo** [reply]
Did you create a project first? If there is no project open, most menu entries are disabled.

When you first open the app, a dialog should pop up asking you to open a project, or if none exist, create a new one.

**Reply by gostone** [reply]
It has sorted itself out somehow. Working now. Thanks for your help

**Reply by vkbo** [reply]
Great! Please mark this as resolved then. There is a checkmark button at the top of my first comment.

**Reply by gostone** [reply]
done
